samples/bluetooth/st_ble_sensor/src/button_svc.c

```diff
@@ -0,0 +1,95 @@
+/** @file
+ *  @brief Button Service sample
+ */
+
+/*
+ * Copyright (c) 2019 Marcio Montenegro <mtuxpe@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/printk.h>
+#include <sys/byteorder.h>
+#include <zephyr.h>
+#include <drivers/gpio.h>
+#include <logging/log.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+#include "button_svc.h"
+
+LOG_MODULE_REGISTER(button_svc);
+
+/* GPIO button; change this to use another GPIO port */
+#ifndef DT_ALIAS_SW0_GPIOS_CONTROLLER
+#ifdef SW0_GPIO_NAME
```

Comment:
Please remove the `SW0_GPIO_NAME` part.
For now, all boards should use gpio definition through dts.

Reply:
done